spmd/compiler/fusion.py

```diff
@@ -2,21 +2,26 @@
 from copy import deepcopy
 from dataclasses import dataclass, field
 from enum import Enum
-from typing import Iterable, List, Optional
+from functools import partial
+from typing import Dict, Iterable, List, Optional
 
 import torch
+import torch.distributed as dist
 import torch.fx as fx
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch.fx.passes.shape_prop import TensorMetadata
 
 from .graph_utils import (
-    OP,
-    get_node_tensor_numel,
+    OP,  # get_all_nodes_of_type,; pretty_print_graph,
```

Comment:
what is this comment about?

Reply:
oh formatter must have mushed that up there.  I've removed completely as not needed.